### PR TITLE
feat: add basic visualisation links for jobs

### PR DESCRIPTION
**Pull request by madwort**

### CODE REVIEW

File: jobserver/views/jobs.py [hunk 1]
```diff
@@ -49,11 +52,22 @@ def get(self, request, *args, **kwargs):
             request.user, "job_cancel", project=job.job_request.workspace.project
         )
 
+        honeycomb_can_view_links = has_role(self.request.user, CoreDeveloper)
+
+        honeycomb_context_starttime = job.created_at - timedelta(minutes=1)
+
+        honeycomb_context_endtime = timezone.now()
+        if job.completed_at is not None:
+            honeycomb_context_endtime = job.completed_at + timedelta(minutes=3)
```
Comment (ghickman) [inline jobserver/views/jobs.py:61]: Why completed_at + 3 minutes here?

**Review by ghickman — APPROVED** [review]
Looks good.  Couple of comments on the timedeltas, might be easier to put the explanations in as comments for the next reader too?

File: jobserver/views/jobs.py (hunk 1 above)
Comment (madwort) [reply jobserver/views/jobs.py:66]: Good question, I don't remember now why this is longer than the starttime - would it be more aesthetic to make them match?

Comment (madwort) [reply jobserver/views/jobs.py:66]: (that they're different makes it look like they're different for a reason, if they're not, then, I should change it!)

File: jobserver/views/jobs.py [hunk 2]
```diff
@@ -49,11 +52,22 @@ def get(self, request, *args, **kwargs):
             request.user, "job_cancel", project=job.job_request.workspace.project
         )
 
+        honeycomb_can_view_links = has_role(self.request.user, CoreDeveloper)
+
+        honeycomb_context_starttime = job.created_at - timedelta(minutes=1)
```
[the remark this answers is not in the document]
Comment (ghickman) [reply jobserver/views/jobs.py:62]: This is great 👍

**Pull request merged**